app/src/wavevr/cpp/DeviceDelegateWaveVR.cpp

```diff
@@ -28,6 +28,7 @@
 #include <wvr/wvr_overlay.h>
 #include <wvr/wvr_system.h>
 #include <wvr/wvr_events.h>
+#include <Controller.h>
```

Comment:
This shouldn't be included in the Device Delegates. If you move the consts to Device.h you won't need to include it from what I can tell.